Coolant be gone

HueyHiggins opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

This will remove the bloodstream and "coolant" blood from IPC's. While it was a neat visual, it caused an almost comical amount of coolant to leak all over the station when an IPC was severely damaged, resulting in lakes of the stuff. What's more, there was no way to stop an IPC from "bleeding" save for burn attacks. Finally, since they had a blood count, this would cause IPC's to stutter even after healed to optimal health, due to their blood not healing fast enough. All and all, something worth exploring down the road, but not right now. 

---

# TODO

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's good to leave future references
-->

- [ ] Task
- [x] Completed Task

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>

![Example Media Embed](https://example.com/thisimageisntreal.png)

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- tweak: Removed the yaml code (lines: 145-155) pertaining to blood/coolant in: deep-station-14/resources/prototypes/entities/mobs/player/ipc.

HueyHiggins committed:
No more coolant

VMSolidus reviewed (changes requested):
It's pretty much guaranteed that the bugs you're describing are fixable via yml changes by just looking into what component is responsible for making mobs eventually stop bleeding and recover blood, and just putting that on them.

Also this PR is neglecting to "fix" the same issue on the Species Swap: Shell

HueyHiggins committed:
Resolved issue

Commented out bleed code instead of deletion. Compromised removing coolant in exchange for fixing Shells and giving them the mechanic instead.
